app/views/appropriate_bodies/participants/index.html.erb

```diff
@@ -1,2 +1,62 @@
-<h1 class="govuk-heading-xl">Participants</h1>
-<h1 class="govuk-heading-xl"><%= @appropriate_body.name %></h1>
+<span class="govuk-caption-xl">Appropriate body</span>
+<h1 class="govuk-heading-xl"><%= @appropriate_body.name %> Participants</h1>
+
+<p class="govuk-body govuk-!-text-align-right">
+  <%= govuk_link_to "Download (csv)", appropriate_body_participants_path(params.permit(:query, :role, :academic_year, :status).merge({format: :csv})) %>
+</p>
+
+<%= render SearchBox.new(
+  query: params[:query],
+  title: "Search",
+  hint: "For example, school, participant name or teacher reference number (TRN)",
+  filters: [
+    {
+      field: :role,
+      value: params[:role],
+      options: @filter.role_options,
+    },
+    {
+      field: :academic_year,
+      value: params[:academic_year],
+      label: "academic year",
+      options: @filter.academic_year_options,
+    },
+    {
+      field: :status,
+      value: params[:status],
+      label: "status",
+      options: @filter.status_options,
+    },
+  ],
+) %>
+
+<table class="govuk-table">
```

Comment:
Could this have been made easier using `govuk_table`?

https://govuk-components.netlify.app/components/table/#building-a-table-from-arrays

The only difference is that it doesn't support adding a 'no results' row that spans the whole table, but from an accessibility point of view we should probably not be rendering a table at all by that point.

Reply:
@peteryates will take a look

Reply:
Cheers @mooktakim. If it's a big job feel free to skip it this time, just wanted to let you know it's there and already included in the project 👍🏽